### ISSUE TITLE
Review Radio Accessibility

### ISSUE THREAD

**Issue by StephenRiosDev** [AlaskaAirlines/auro-formkit#457]
### Description

Assess accessibility of the Radio component, including using them in a radio group. Ensure it meets all W3C standards and that the screen reader functions as expected.

Note: Individual radio buttons should function the same as a normal HTML radio button would.

W3C Example (Radio Group): https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio/

### Exit criteria (To be determined by the Auro team)

Radio component should meet or exceed all W3C 2.0 AA criteria.

**Comment by blackfalcon** [comment]
> Radio component should meet or exceed all W3C 2.0 AA criteria.

The floor-wide standard is to meet or exceed WCAG 2.2 Level AA. Please see [Alaska Airlines Digital Accessibility and Inclusion Policy](https://wiki.devtools.teamaag.com/standards/accessibility/digital-accessibility-policy) for more information.
